aptos-move/e2e-testsuite/src/tests/failed_transaction_tests.rs

```diff
@@ -26,6 +26,7 @@ use move_deps::move_core_types::{
 fn failed_transaction_cleanup_test() {
     test_with_different_versions! {CURRENT_RELEASE_VERSIONS, |test_env| {
         let mut executor = test_env.executor;
+        // TODO(Gas): double check this
```

Comment:
Do you still need this?

Reply:
Yes. Otherwise I would end up spending hours to find these numbers when I inevitably have to change them in the future.